infra/syntax/Json.cpp

```diff
@@ -38,15 +38,21 @@ namespace infra
 
             switch (*next++)
             {
-                case '"':  return '"';
-                case '\\': return '\\';
-                case 'b': return '\b';
-                case 'f': return '\f';
-                case 'n': return '\n';
-                case 'r': return '\r';
-                case 't': return '\t';
-                case 'u':
-                {
+                case '"':
```

Comment:
Hmm this is unfortunate. But apparently clang-format cannot put single-return-statements on the same line as a case statement, without affecting formatting other case statements.